clients/web/src/presentation/router/protectedRoutes.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { Navigate, Outlet } from 'react-router-dom';
+import useFetchData from '@client-commons/hooks/useFetchData';
+import { getUser } from '@client-commons/platform/aws/auth/awsAuth';
+import FullPageLoader from '../components/loader/FullPageLoader';
+import * as RouteConsts from '../../constants/routeConsts';
+import DefaultLayout from '../layout/DefaultLayout/DefaultLayout';
+
+export function ProtectedRoute() {
+  const [fetchUser, loading, user, ] = useFetchData(getUser);
```

Comment:
any reason why we are not showing the error?

Reply:
i forgot about checking the error here. updated in a23931bef884bc8c5359f5160309c4f34b8afa37